src/harness/unittests/extractConstants.ts

```diff
@@ -0,0 +1,80 @@
+/// <reference path="extractTestHelpers.ts" />
+
+namespace ts {
+    describe("extractConstants", () => {

```

Comment:
>extractConstants [](start = 14, length = 16)

This test coverage isn't great because I didn't want to revise everything as soon as I updated the insertion locations.